site/quorum-queue.md

```diff
@@ -0,0 +1,267 @@
+<!--
+Copyright (c) 2007-2018 Pivotal Software, Inc.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Quorum Queue NOSYNTAX
+
+## Overview
+
+The quorum queue is a new (since 3.8) queue type for RabbitMQ implementing a persistent,
+replicated fifo queue using [the Raft consensus algorithm](https://raft.github.io/).
+
+The quorum queue type is an alternative to persistent [highly available (mirrored)
+queues](ha.html) (HA queues) for use cases where data
+safety is paramount.
+
+#### [Motivation](#motivation)
+
+Existing HA queues have technical limitations that makes it difficult to provide
+comprehensible guarantees and simple failure semantics within RabbitMQ.
+
+Certain network partitions / failure scenarios can result in HA queues
+losing confirmed messages. Quorum queues are designed to provide simpler
+semantics that users should find easier to reason about when designing
+and operating their systems.
+
+##  Usage
+
+#### [Declaring](#declaring)
+
+To declare a quorum queue set the `x-queue-type` queue argument to `quorum`
+(the default is `classic`). This will declare a quorum queue with a member on
+every currently configured RabbitMQ node.
+After that the queue can be bound to any exchange just as any other
+RabbitMQ queues.
+
+It can also be created through the management UI using the queue type drop down
+menu.
+
+#### [Controlling the quorum queue size](#controlling-size)
```

Comment:
The `quorum-queue-size` name may be confused with a queue depth. Maybe it can be renamed to reflect the fact that this is a limit of nodes members of the queue cluster?